core/modules/editor/types/date.js

```diff
@@ -0,0 +1,27 @@
+/*\
+title: $:/core/modules/editor/types/date.js
+type: application/javascript
+module-type: inputtype
```

Comment:
Perhaps the module type should be `editmapping`? (the word "type" is a bit overloaded in the codebase)